### PR TITLE
Feature/v4

### PR DESCRIPTION
**Pull request by twanariens**
A big CSS overhaul.

### CODE REVIEW

File: src/Ombi/ClientApp/src/app/settings/emby/emby.component copy.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+﻿
```
Comment (tidusjar) [inline src/Ombi/ClientApp/src/app/settings/emby/emby.component copy.html:1]: Is this file needed?

Comment (twanariens) [reply src/Ombi/ClientApp/src/app/settings/emby/emby.component copy.html:1]: The copy is there as a failsafe. I made changes to the emby.component.html to make it use mat-tab-group. But I don't have Emby running so I am unable to check if it is working as intended.
